modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/services/FileManagerImport.java

```diff
@@ -82,7 +88,7 @@ public DocumentModel run(Blob blob) throws OperationException, IOException {
         DocumentModel currentDocument = getCurrentDocument();
         String path = currentDocument.getPathAsString();
         FileImporterContext fileCreationContext = FileImporterContext.builder(session, blob, path)
-                                                                     .overwrite(overwite)
+                                                                     .overwrite(overwrite)
```

Comment:
I think we want `overwite || overwrite` here?

Reply:
And it misses some unit tests to validate that both parameters work as expected :)